### PR TITLE
[zh] Rework stateful application tasks

### PR DESCRIPTION
**Pull request by tengqm**
English source was lost and there are punctuation and space nits.

### CODE REVIEW

File: content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 1]
```diff
@@ -1,131 +1,188 @@
...
 
-
-* 在环境中通过磁盘创建一个PersistentVolume.
+<!--
+* Create a PersistentVolume referencing a disk in your environment.
+* Create a MySQL Deployment.
+* Expose MySQL to other pods in the cluster at a known DNS name.
+-->
+* 在你的环境中创建一个引用磁盘的 PersistentVolume
 * 创建一个MySQL Deployment.
```
Comment (tanjunchen) [inline content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md:24]:
```suggestion
* 创建一个 MySQL Deployment.
```

File: content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 2]
```diff
@@ -1,131 +1,188 @@
...
-* 在集群内以一个已知的 DNS 名将 MySQL 暴露给其他 pods.
-
-
-
+* 在集群内以一个已知的 DNS 名将 MySQL 暴露给其他 Pod
```
Comment (tanjunchen) [inline content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md:25]:
```suggestion
* 在集群内以一个已知的 DNS 名称将 MySQL 暴露给其他 Pod。
```

**Review by tanjunchen — APPROVED** [review]
/lgtm
thanks

File: content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md:25]: Done.

**Pull request merged**